content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +200,80 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before
+continue in the following recommendations, make sure you understand its options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path for patch release (bug fixes)
+
+Within a channel each patch release should be directly upgradable to the `head channel` from the latest version shipped 
+in the minor channel behind. Use skips or skipRange to provide this behaviour. (i.e. using the skipRange in the 
+`stable-3.6` patch for `3.5.<latest>: >= 3.5.<first z of the minor channel>. < 3.6.<new pacth release>`)
+
+#### Example
+
+Let's imagine that you will publish the Operator bundle `3.6.30` (patch release under the channel `stable-3.6`) and
+that you have published so far:
+
+- Operator bundles versions from `3.5.0` to `3.5.25` under the `stable-3.5` channel
+- Operator bundles versions from `3.6.0` to `3.6.29` under the `stable-3.6` channel
+
+Then, in this case, your newer patch release `3.6.30` would be configured with `skipRange: >=3.5.25 < 3.6.30` in
+order to only supports upgrading to the newest `3.6.z` from a pretty recent `3.5.z`
+
+**Note** If you would like to ensure that the users of your Operator still able to install the Operator bundle version
+`3.6.29` by using the option `startingCSV` added manually in the subscription then, you will need to also use the `replaces: 3.6.29`
+for this Operator bundle does not get pruned from the index catalog via the OLM resolver.
+
+#### Attention (Be aware of the following example scenario)
+
+If you have the channel `stable-3.7` where the head of channel is `3.7.10` and then, you provide a new patch release
+with a bug fix using the Operator bundle version `3.7.11` then, imagine that you configure your new publication to skip all
+that was published under the channel `stable-3.7` (e.g. skipRange: `">= 3.7.0 < 3.7.11"` replaces: `3.6.z <latest release on 3.6 channel>` ).
+
+You could ensure that your users can move forward easier to the latest version since in this range `">= 3.7.0 < 3.7.11"`
+as they will be able to install the new `3.7.11` release from  `3.6.z <latest release on 3.6 channel>` directly 
+instead of get installed first the Operator version `3.7.0` which we know that have bug fixes in place already.
+
+However, **attention** that means you would no longer able to provide any patch release to the minor channel `stable-3.6`.    
+Note that if you publish `3.6.latest+1` when this version comes out, and your users upgrades to it, they will have no way to
+upgrade from `3.6.latest+1` to any solution published under the channel `stable-3.7`.
```

Comment:
suggest this instead for the above paragraph:


However, that means that if/when you publish a `3.6.latest+1` and a user upgrades to it, they will not
have a path to `3.7` because `3.6.latest+1` is not included in the skipRange defined by any existing `3.7.z` bundle.  You would have to then publish a new `3.7.z` bundle that expands the skipRange to include 3.6.latest+1 before users would be able to upgrade to a 3.7.z version.

Reply:
(this is a restriction that becomes less problematic with FBC when operator authors are able to update the skipRange of a previously published bundle, so they could extend the skipRange of 3.7.11 to include 3.6.latest+1, but of course we aren't there yet)

Reply:
I think a follow up here will be to change this doc for FBC or tweak it. 
IMO I do not think that we should only FBC instructions any way since it will be optional in the future for the authors and many of them might never adopt it.

Reply:
>  I do not think that we should only FBC instructions any way since it will be optional in the future for the authors and many of them might never adopt it.

at a minimum i agree it's probably too early to do so.

Reply:
this is reminiscent of the discussion we had many months ago when we replaced some imperative `opm registry add` docs with the new FBC docs.

the main branch of the upstream docs site focuses on FBC, not the deprecated sqlite workflows.

Reply:
to reframe this as a question, why should we recommend that upstream users keep using deprecated workflows when FBC exists and helps solve a bunch of these problems?

if there are pipelines that have not adopted FBC, those pipelines likely need to have documentation about their requirements and assumptions.

IMO the upstream OLM docs site should help catalog maintainers and pipeline builders understand how to get off the deprecated path and onto FBC.

Reply:
Closing this one since we spoke shows that we agree that its goal is only to add what is missing we probably want to work in the future to see how we can shape it for FBC.